config/dev/remote-clusterdeployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: ClusterDeployment
+metadata:
+  name: remote-${CLUSTER_NAME_SUFFIX}
+  namespace: ${NAMESPACE}
+spec:
+  template: remote-cluster-0-1-0
+  credential: remote-cred
+  propagateCredentials: false
+  config:
+    machines:
+      - address: ${MACHINE_0_ADDRESS}
+        user: root
+        port: 22
+      - address: ${MACHINE_1_ADDRESS}
+        user: root
+        port: 22
+  serviceSpec:
+    services:
+      - template: kyverno-3-2-6
+        name: kyverno
+        namespace: kyverno
+      - template: ingress-nginx-4-11-0
+        name: ingress-nginx
+        namespace: ingress-nginx
```

Comment:
Is it necessary to have these services in the managed cluster?

Reply:
No, not really. Let's keep this dev ClusterDeployment as simple as possible.